### DISCUSSION TITLE
Follow mouse cursor

### DISCUSSION THREAD

**Discussion by MrKinau** [rustdesk/rustdesk#2059] [General]
An option to follow the mouse cursor could be very useful for multi monitor setups.
RustDesk should automatically switch the screen to the remotes mouse cursor position.

**Comment by MinecraftiUndCo** [comment]
That would be a pretty good feature I guess. Some of my friends have 3 monitors and I dont want to switch always manually between them. Please implement this feature! 👍

**Comment by Summerfeeling** [comment]
That should have highest priority!
